### DISCUSSION TITLE
MRF 2.0.0 Methodology = "other"

### DISCUSSION THREAD

**Discussion by CraneBarry** [CMSgov/hospital-price-transparency#90] [Q&A] [answered]
![image](https://github.com/CMSgov/hospital-price-transparency/assets/157032880/9405f83d-344f-4451-8d2f-769fe766c040)
![Other Additional Payor Note Scenario 20240423_a](https://github.com/CMSgov/hospital-price-transparency/assets/157032880/d0e8ff2d-0f65-439c-b7c5-6a08ae9335bb)

**Comment by RidenourK** [comment, marked as answer]
@CraneBarry. Thank you for your question. 

Could you please provide a few sample rows of this MRF that include the codes that correspond to the rows in your screenshot, or a sample of the codes themselves with any corresponding standard charge information to the CMS HPT mailbox at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov)?
